src/components/mapPageComponents/mapView/index.tsx

```diff
@@ -193,57 +193,46 @@ const MapView: React.FC<MapViewProps> = ({ blocks, neighborhoods }) => {
         const neighborhoodsLayer = new google.maps.Data({ map });
 
         // Function: Returns name in shortHand
-        /*
-        function shortHand(name: string): string {
-          for (const shName of shortHandNames) {
-            if (shName.key === name) {
-              return shName.value;
-            }
-          }
-          return name;
-        }
-        */
+        function shortHand(name:string, shortHandNames:{ [key:string]: string }) {
```

Comment:
💭 For testing purposes, it might be better to put this function in a separate file, either a new one or `stringFormat.tsx`   (then you can also just put the test in `stringFormat.test.ts`). It's also a pretty general function now since you could pass in any dictionary, so consider renaming it.